tests/api2/activedirectory.py

```diff
@@ -8,29 +8,25 @@
 from time import sleep
 apifolder = os.getcwd()
 sys.path.append(apifolder)
-from auto_config import pool_name, ip, hostname, scale, user, password
-from functions import GET, POST, PUT, DELETE, SSH_TEST, wait_on_job
+from auto_config import pool_name, ip, hostname, user, password
+from functions import (
+    GET,
+    POST,
+    PUT,
+    DELETE,
+    SSH_TEST,
+    cmd_test,
+    wait_on_job
+)
 
 try:
     from config import AD_DOMAIN, ADPASSWORD, ADUSERNAME, ADNameServer
+    AD_USER = fr"AD01\{ADUSERNAME.lower()}"
```

Comment:
You can probably remove `fr` from these too

Reply:
flake8 will be annoying if I remove `r`. My editor gives me W605  https://www.flake8rules.com/rules/W605.html

Reply:
Also, pytest complain about that.